src/olympia/activity/utils.py

```diff
@@ -205,34 +205,52 @@ def log_and_notify(action, comments, note_creator, version, perm_setting=None,
         task_user = set()
     # Collect reviewers on the thread (excl. the email sender and task user for
     # automated messages).
-    reviewers = ((log_users | staff_cc) - addon_authors - task_user -
-                 {note_creator})
+    reviewers = (log_users - addon_authors - task_user - {note_creator})
+    staff_cc = (staff - reviewers)
```

Comment:
you need to `- addon_authors - {note_creator}` too or you'll get dupes for add-ons you're a developer on and your own replies.